src/Application/Actions/ActionError.php

```diff
@@ -67,6 +64,10 @@ public function setDescription(?string $description = null): self
         return $this;
     }
 
+    #[ArrayShape([
```

Comment:
This is cool! But isn't `type` a string as above?

Reply:
Good catch